vulture_os/workflow/config/haproxy_portal.conf

```diff
@@ -122,35 +120,47 @@ frontend Workflow_{{conf.id}}
 
     ### ACCESS ###
     # Workflow {{conf.name}}
+
+    # get header part of the JWT
+    http-request set-var(txn.alg) http_auth_bearer,jwt_header_query('$.alg')
+
+    # get payload part of the JWT
+    http-request set-var(txn.iss) http_auth_bearer,jwt_payload_query('$.iss')
+    http-request set-var(txn.aud) http_auth_bearer,jwt_payload_query('$.aud')
+    http-request set-var(txn.exp) http_auth_bearer,jwt_payload_query('$.exp','int')
+    http-request set-var(txn.now) date()
```

Comment:
need to add a jinja condition to calculate those values only when necessary (JWT validation activated on this workflow)

Reply:
I guess this comment was written before the last commit as we discussed earlier

Reply:
The solution I provided doesn't look very nice to me, feel free to suggest something else